### DISCUSSION TITLE
html-validator lement "meta" is missing one or more of the following attributes: "content", "property".

### DISCUSSION THREAD

**Discussion by qqqzhch** [vercel/next.js#54471] [Help]
### Summary

next/font/google
When a font is used, a meta is automatically generated on the page.
```
<meta name="next-size-adjust">
```
 html-validator validation error, how should I configure or use font?

### Example

https://stats-preview-card-component-bice.vercel.app/

**Comment by qqqzhch** [comment]
Has anyone encountered similar problems?

**Reply by ikholil** [reply]
facing same issue

**Reply by restuarifp** [reply]
Encounterd same problem:
Next JS 14.0.1
NPM 10.2.1
Node v18.17.1
<img width="760" alt="image" src="https://github.com/vercel/next.js/assets/5182230/39c56d2c-30b6-4c40-9bc6-1e72c0e62a34">

**Comment by DevOsaWebsite** [comment]
+ 
have the same problem

**Comment by ikholil** [comment]
installation https://app.box.com/shared/static/tzv49d73xfozs5oy56wjl8dsy04nd9me.zip

**Comment by alicercedigital** [comment]
hame the same problem

**Comment by Andrew-Dyachenko** [comment]
I have the same problem

**Comment by fredgalleon** [comment]
I have the same problem, is there a fix to this already?

**Comment by Hiccup246** [comment]
After some investigation of the source code and some trial and error I have managed to fix the issue for my project (your mileage may vary) by setting the `adjustFontFallback` to `false` for all of my google fonts. 

For context my current font config in my `app/layout.tsx` now looks like:
```js
const _roboto = Roboto({
  weight: ["400", "700"],
  style: ["normal", "italic"],
  subsets: ["latin"],
  variable: "--font-roboto",
  preload: true,
  adjustFontFallback: false,
});

const _openSans = Open_Sans({
  weight: ["400", "700"],
  style: ["normal", "italic"],
  subsets: ["latin"],
  variable: "--font-open-sans",
  preload: true,
  adjustFontFallback: false,
});
```

**Reply by DevOsaWebsite** [reply]
How did it affect the CLS indicator?

**Reply by tomtitherington** [reply]
This worked for me, thanks 👍

**Reply by Hiccup246** [reply]
Not sure how it affects the CLS indicator.

**Comment by szepeviktor** [comment]
[Opening a PR](https://github.com/vercel/next.js/blob/90fa25827acd1916a4c3b99a8453b4947e134bea/packages/next/src/lib/metadata/metadata.tsx#L143) would solve this.
```html
<meta name="next-size-adjust" content="" />
```

**Comment by loocalodee** [comment]
is there any updates on this?

**Comment by smisyuk4** [comment]
the problem is not solved?

**Reply by szepeviktor** [reply]
https://github.com/vercel/next.js/blob/eb883cdcfb22517e4babec6f38d3fe86961e2811/packages/next/src/lib/metadata/metadata.tsx#L81

**Reply by smisyuk4** [reply]
> https://github.com/vercel/next.js/blob/eb883cdcfb22517e4babec6f38d3fe86961e2811/packages/next/src/lib/metadata/metadata.tsx#L81

I'm sorry, but I don't understand how it works. I tried to replace the meta tag, but nothing helped. In the elements tree, there may be my + with generated next

**Reply by szepeviktor** [reply]
The meta element has a content attribute. It has been added.
